code/modules/mob/living/carbon/human/species_types/IPC.dm

```diff
@@ -0,0 +1,114 @@
+/datum/species/IPC // im fucking lazy mk2 and cant get sprites to normally work
```

Comment:
Why is it capitalized?

Reply:
Because it's such a cool feature he has to shout its name

Reply:
because it can be and i had capslock on : ^ )